OWASP/SecureTea-Project
# Web deface set

## Status
**READY**

## Description
Added set based deface detection to find what is actually defaced in the server

## Related PRs
None

## Todos


## Deploy Notes
None

## Steps to Test or Reproduce

```sh
git pull --prune
git checkout <feature-branch>
```

1. Run the deface detection pointing to your apache/nginx server and try changing a file.
2. Check log file

## Impacted Areas in Application
List general components of the application that this PR will affect:

* Web deface detection
